libexec/dss/deploy-ilk-usdc

```diff
@@ -26,3 +26,4 @@ fi
 
 # shellcheck source=libexec/dss/deploy-ilk
 . "$LIBEXEC_DIR/dss/deploy-ilk" "USDC" "${1:-"A"}" "GemJoin5"
+. "$LIBEXEC_DIR/dss/deploy-ilk" "USDC" "${1:-"B"}" "GemJoin5"
```

Comment:
This line actually shouldn't be added. The USDC-B will be deployed just with what you added in the config files. `${1:-"A"}` just indicates to use `A` if `$1` is not set (a legacy thing when this script was in `dss-deploy`).